packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/auth/resource.ts

```diff
@@ -1,5 +1,9 @@
 import { defineAuth, secret } from '@aws-amplify/backend';
-import { defaultNodeFunc } from '../function.js';
+import { defaultNodeFunc, funcWithSchedule } from '../function.js';
+
+const customEmailSenderFunction = {
+  handler: funcWithSchedule,
+};
```

Comment:
Related to my previous comments.

We shouldn't reuse `funcWithSchedule` here. Because we'll be blending together two independent scenarios.

We should create a function similar to `funcWithSchedule` with it's own associated queue. I.e. create similar but isolated setup.